espnet2/spk/espnet_model.py

```diff
@@ -79,12 +79,18 @@ def forward(
             extract_embd: a flag which doesn't go through the classification
                 head when set True
             spk_labels: (Batch, )
+            task_tokens: (Batch, )
```

Comment:
@Can you explain what `spk_labels` and `task_tokens` represent in the docstring?